### PR TITLE
Add claude

### PR DESCRIPTION
**Pull request by jtruong-coursera**
Adds Claude LLM to Playground using Anthropic's Bedrock SDK, `@anthropic-ai/bedrock-sdk`. It gains access to the Claude LLM via an IAM role with policies to access AWS Bedrock

### CODE REVIEW

**Review by dguo-coursera — APPROVED** [review]

File: types/claude.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+};
+
+export class ClaudeAIError extends Error {
+  type: string;
+  param: string;
+  code: string;
+
+  constructor(message: string, type: string, param: string, code: string) {
+    super(message);
+    this.name = 'OpenAIError';
```
Comment (dguo-coursera) [inline types/claude.ts]: Should this be BedrockError or ClaudeAIError?

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 5431",
```
Comment (saakshaat-singh) [inline package.json:6]: do we need to specify the port?

File: types/claude.ts (hunk 1 above)
Comment (jtruong-coursera) [reply types/claude.ts]: Ah yes, changing to `ClaudeAIError`, thanks :)

**Review by saakshaat-singh — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (jtruong-coursera) [reply package.json:6]: @dguo-coursera would know better. This is from uncommitted changes before I implemented Claude

**Pull request merged**